packages/components/transfer/src/index.vue

```diff
@@ -183,14 +185,14 @@ export default defineComponent({
       emit
     )
 
-    const leftPanel = ref(null)
-    const rightPanel = ref(null)
+    const leftPanel = ref<TransferType>()
+    const rightPanel = ref<TransferType>()
 
     const clearQuery = (which: 'left' | 'right') => {
       if (which === 'left') {
-        leftPanel.value.query = ''
+        leftPanel.value!.query = ''
       } else if (which === 'right') {
-        rightPanel.value.query = ''
+        rightPanel.value!.query = ''
```

Comment:
Can we use `switch` in this case?

Reply:
I would also like to ask you to add a test against this, so we can assure this code won't be wrong again.

Reply:
> Can we use `switch` in this case?  

ok.

> I would also like to ask you to add a test against this, so we can assure this code won't be wrong again.  

Good advice.